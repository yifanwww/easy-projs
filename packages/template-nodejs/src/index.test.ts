import * as package_exports from './index';

test('Test all exports from this package', () => expectSnapshot(Object.keys(package_exports)));

test('Test function `templateNodejs`', () => expectSnapshot(package_exports.templateNodejs()));
