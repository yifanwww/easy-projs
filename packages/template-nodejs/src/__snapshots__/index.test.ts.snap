// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test all exports from this package 1`] = `
Array [
  "templateNodejs",
]
`;

exports[`Test function \`templateNodejs\` 1`] = `"This package named \`@easy-packages/template-nodejs\` is used for creating new nodejs packages."`;
